go/genesis/tendermint.go

```diff
@@ -0,0 +1,146 @@
+package genesis
```

Comment:
Uh, this doesn't seem nice, can we move this Tendermint-specific thing somewhere in `go/tendermint`? Also it seems strange that this is a genesis provider -- as providers are meant for loading genesis documents but this seems like it is used for dumping one?

Maybe instead, as I mentioned in my earlier comment, expose a method in `consensus.Backend` (e.g., `GetGenesis(int64)` and implement it in `go/tendermint` instead.

Reply:
makes sense :+1: